tests/pipelines/test_pipelines_text_to_audio.py

```diff
@@ -39,37 +39,59 @@ class TextToAudioPipelineTests(unittest.TestCase):
     model_mapping = MODEL_FOR_TEXT_TO_WAVEFORM_MAPPING
     # for now only test text_to_waveform and not text_to_spectrogram
 
+    @require_torch
+    def test_tiny_musicgen_pt(self):
+        music_generator = pipeline(
+            task="text-to-audio",
+            model="hf-internal-testing/tiny-random-MusicgenForConditionalGeneration",
+            framework="pt",
+        )
```

Comment:
I might be wrong, but models that are in `hf-internal-testing` should be tested with `run_pipeline_test` called by [`PipelineTesterMixin`](https://github.com/huggingface/transformers/blob/32ec7345f2d752c294ddf5aff495b657c9cd9d3b/tests/test_pipeline_mixin.py#L122), right ?

Maybe @ydshieh can answer this question ?

Reply:
Good spot - MusicGen currently isn't tested by the pipeline, but I'll fix this in a follow-up PR